packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -122,6 +122,11 @@ const propTypes = {
    * If the `variant` prop if set to `12-hour` for one of these supported locales, the variant will be ignored and defaults to `24-hour`.
    */
   timeVariant: PropTypes.oneOf([DateTimeUtils.FORMAT_12_HOUR, DateTimeUtils.FORMAT_24_HOUR]),
+  /**
+   * Timezone value to indicate in which timezone the date-time component is rendered.
+   * The value provided should be a valid [timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) string, else will default to browser/local timezone.
+   */
+  timeZone: PropTypes.string,
 };
 
 const defaultProps = {
```

Comment:
Should this add the default timeZone value? `timeZone: moment.tz.guess()`?

Reply:
I think we should so that we only need to check for the validity of the timezone and not the existence of it in some of the helpers in DateTimieUtil.